driver/state/state_test.go

```diff
@@ -19,7 +20,9 @@ type DriverStateTestSuite struct {
 
 func (s *DriverStateTestSuite) SetupTest() {
 	s.ClientTestSuite.SetupTest()
-	state, err := New(context.Background(), s.RPCClient)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond*200)
```

Comment:
Not sure if use `time.Millisecond*200` is a good choice here, shall we just manually cancel the `ctx` here?